examples/elasticsearch/README.md

```diff
@@ -97,34 +96,646 @@ Did it work?
-[2015-08-30 10:01:44,010][INFO ][http                     ] [Hammerhead] bound_address {inet[/0:0:0:0:0:0:0:0:9200]}, publish_address {inet[/10.244.48.2:9200]}
-[2015-08-30 10:01:44,011][INFO ][node                     ] [Hammerhead] started
-[2015-08-30 10:01:44,042][INFO ][gateway                  ] [Hammerhead] recovered [0] indices into cluster_state
-[2015-08-30 10:08:02,517][INFO ][cluster.service          ] [Hammerhead] added {[Tenpin][2gv5MiwhRiOSsrTOF3DhuA][es-78e0s][inet[/10.244.54.4:9300]]{master=true},}, reason: zen-disco-receive(join from node[[Tenpin][2gv5MiwhRiOSsrTOF3DhuA][es-78e0s][inet[/10.244.54.4:9300]]{master=true}])
-[2015-08-30 10:10:10,645][INFO ][cluster.service          ] [Hammerhead] added {[Evilhawk][ziTq2PzYRJys43rNL2tbyg][es-rjmer][inet[/10.244.33.3:9300]]{master=true},}, reason: zen-disco-receive(join from node[[Evilhawk][ziTq2PzYRJys43rNL2tbyg][es-rjmer][inet[/10.244.33.3:9300]]{master=true}])
-```
+[2015-11-20 18:27:53,449][INFO ][node                     ] [Danielle Moonstar] version[2.0.0], pid[13], build[de54438/2015-10-22T08:09:48Z]
+[2015-11-20 18:27:53,456][INFO ][node                     ] [Danielle Moonstar] initializing ...
+[2015-11-20 18:27:53,786][INFO ][plugins                  ] [Danielle Moonstar] loaded [cloud-kubernetes], sites []
+[2015-11-20 18:27:53,823][INFO ][env                      ] [Danielle Moonstar] using [1] data paths, mounts [[/data (/dev/disk/by-uuid/7c2ba6f8-3e2f-49da-9d24-211659759bdb)]], net usable_space [90gb], net total_space [98.3gb], spins? [possibly], types [ext4]
+[2015-11-20 18:27:56,545][INFO ][node                     ] [Danielle Moonstar] initialized
+[2015-11-20 18:27:56,551][INFO ][node                     ] [Danielle Moonstar] starting ...
+[2015-11-20 18:27:56,612][INFO ][transport                ] [Danielle Moonstar] publish_address {10.56.0.21:9300}, bound_addresses {10.56.0.21:9300}
+[2015-11-20 18:27:56,622][INFO ][discovery                ] [Danielle Moonstar] myesdb/C9nmBJw3TJ22JcAVQmdNeg
+[2015-11-20 18:28:01,571][INFO ][cluster.service          ] [Danielle Moonstar] new_master {Danielle Moonstar}{C9nmBJw3TJ22JcAVQmdNeg}{10.56.0.21}{10.56.0.21:9300}{master=true}, reason: zen-disco-join(elected_as_master, [0] joins received)
+[2015-11-20 18:28:01,614][INFO ][http                     ] [Danielle Moonstar] publish_address {10.56.0.21:9200}, bound_addresses {10.56.0.21:9200}
+[2015-11-20 18:28:01,615][INFO ][node                     ] [Danielle Moonstar] started
+[2015-11-20 18:28:01,652][INFO ][gateway                  ] [Danielle Moonstar] recovered [0] indices into cluster_state
+[2015-11-20 18:29:36,776][WARN ][http.netty               ] [Danielle Moonstar] Caught exception while handling client http traffic, closing connection [id: 0xf28436c7, /10.56.1.12:43018 => /10.56.0.21:9200]
```

Comment:
@pires Here's what I'm getting following the example literally. I can see it's because discovery happened on 9200; What I don't see, is how do we resolve this in the rc - when the simple example fires up a master running both http and transport services.